en/docs/reference/customize-product/extending-api-manager/extending-workflows/invoking-the-api-manager-from-the-bpel-engine.md

```diff
@@ -16,12 +16,12 @@ Once the workflow configurations are finalized at the BPEL, the call-back URL of
 <tbody>
 <tr class="odd">
 <td>SOAP</td>
-<td><p><a href="https://localhost:8243/services/WorkflowCallbackService" class="uri">https://localhost:8243/services/WorkflowCallbackService</a></p>
-<p>WSDL Location : <a href="http://localhost:8280/services/WorkflowCallbackService?wsdl" class="uri">http://localhost:8280/services/WorkflowCallbackService?wsdl</a></p></td>
+<td><p>https://localhost:8243/services/WorkflowCallbackService</p>
+<p>WSDL Location : http://localhost:8280/services/WorkflowCallbackService?wsdl</p></td>
 </tr>
 <tr class="even">
 <td>REST</td>
-<td><a href="https://localhost:9443/api/am/admin/v2/workflows/update-workflow-status" class="uri">https://localhost:9443/api/am/admin/v2/workflows/update-workflow-status</a></td>
+<td>https://localhost:9443/api/am/admin/v2/workflows/update-workflow-statu</td>
```

Comment:
Please double check, whether this should be "status"

Reply:
Fixed [88fcc77](https://github.com/wso2/docs-apim/pull/7338/commits/88fcc77ee6070584459ce9be3940cd8ad76a3a06)